examples/get_iss_breast_data.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import io
+import os
+from typing import IO, Tuple
+
+from skimage.io import imread, imsave
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Indices
+from starfish.util.argparse import FsExistsType
+
+SHAPE = 1044, 1390
+
+
+class IssCroppedBreastTile(FetchedTile):
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return SHAPE
+
+    @property
+    def format(self) -> ImageFormat:
+        return ImageFormat.TIFF
+
+    @staticmethod
+    def crop(img):
+        crp = img[40:1084, 20:1410]
+        return crp
+
+    def tile_data_handle(self) -> IO:
+        im = self.crop(imread(self.file_path))
+        fh = io.BytesIO()
+        imsave(fh, im, plugin='tifffile')
+        fh.seek(0)
+        return fh
+
+
+class ISSCroppedBreastPrimaryTileFetcher(TileFetcher):
+    def __init__(self, input_dir):
+        self.input_dir = input_dir
+
+    @property
+    def ch_dict(self):
+        ch_str = ['Cy3 5', 'Cy3', 'Cy5', 'FITC']
+        ch = [2, 1, 3, 0]
+        ch_dict = dict(zip(ch, ch_str))
+        return ch_dict
+
+    @property
+    def hyb_dict(self):
+        hyb_str = ['1st', '2nd', '3rd', '4th']
+        return dict(zip(range(4), hyb_str))
+
+    def get_tile(self, fov: int, hyb: int, ch: int, z: int) -> FetchedTile:
+        filename = 'slideA_' + str(fov + 1) + '_' + \
```

Comment:
string concatenation is a huge mess and very hard to read.  consider switching to [f-style python strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings).